test/e2e_node/summary_test.go

```diff
@@ -0,0 +1,283 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gstruct"
+	"github.com/onsi/gomega/types"
+)
+
+// TODO(timstclair): Move this test out of the flaky suite once it has demonstrated stability.
+var _ = framework.KubeDescribe("Summary API [Flaky]", func() {
+	f := framework.NewDefaultFramework("summary-test")
+	Context("when querying /stats/summary", func() {
+		It("should report resource usage through the stats api", func() {
+			const pod0 = "stats-busybox-0"
+			const pod1 = "stats-busybox-1"
+
+			By("Creating test pods")
+			createSummaryTestPods(f, pod0, pod1)
+			// Wait for cAdvisor to collect 2 stats points
+			time.Sleep(15 * time.Second)
+
+			// Setup expectations.
+			const (
+				kb = 1000
```

Comment:
I think it would be `kib` for 1024. In practice it doesn't matter, because the bounds aren't that precise.